plugin/storage/cassandra/savetracetest/main.go

```diff
@@ -44,8 +45,9 @@ func main() {
 	if err != nil {
 		logger.Fatal("Cannot create Cassandra session", zap.Error(err))
 	}
+	tracer := otel.GetTracerProvider()
```

Comment:
it's not initialized, so essentially the same as NoOp. Why don't you call jtracer.New ?